frontend/src/service/Service.js

```diff
@@ -0,0 +1,13 @@
+// TODO(juliaogris): error handling for async request
```

Comment:
I shall never forget again from here onwards (until I forget it).
You make sense to me.
Done.

Reply:
Actually, this TODO is done ;)
Updated the others (Capitalised them as will - like commit titles; )).

Reply:
i personally never liked capitalised titles (note the amount of capitalisation in this and other comments?), but since that's the style of the day, carry on.